src/modules/account/components/overview/index.tsx

```diff
@@ -86,12 +91,12 @@ const Overview = ({ customer, orders }: OverviewProps) => {
                         >
                           <Container className="bg-gray-50 flex justify-between items-center p-4">
                             <div className="grid grid-cols-3 grid-rows-2 text-small-regular gap-x-4 flex-1">
-                              <span className="font-semibold">Date placed</span>
+                              <span className="font-semibold">{t(k.DATE_PLACED)}</span>
                               <span className="font-semibold">
-                                Order number
+                                {t(k.ORDER_NUMBER)}
                               </span>
                               <span className="font-semibold">
-                                Total amount
+                                {t(k.TOTAL_AMOUNT)}
                               </span>
                               <span data-testid="order-created-date">
                                 {new Date(order.created_at).toDateString()}
```

Comment:
`next-intl` has utilities for formatting dates, numbers and more via `useFormatter`. I'd really recommend using those as they will help to align formatting with the given locale.